osu.Game/Users/Region.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Users
+{
+    public class Region : IHasDrawableRepresentation<DrawableFlag>

```

Comment:
i think we should go with `Country`, not `Region`. 99% if not 100% of these are countries.